Sidney-/slugtrade
# Signup

New Functionality:
- Signup page accessible through the home page
- Creates a new user and user profile upon submitting the form and then logs the user in

How to test:
- Log out of your current user, and click Sign Up from the home page
- Fill out the form (all fields are required and the text fields should have a maxlength property which limits the character inputs)
- Submit the form and make sure all of the data was inputted correctly by either viewing it in the admin panel or checking your profile page
- Verify that the merge didn't break profile, edit_profile, or add_closet_item (I tested all of these myself when I did the merge and found no problems, but it would be good to give it a second set of eyes)

Notes:
- Sidney did a bug fix here which could end up breaking some of Shane's stuff when db-populate is merged
